app/cdap/components/GridTable/components/NestedMenu/index.tsx

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { Menu } from '@material-ui/core';
+import MenuComponent from 'components/GridTable/components/MenuComponent';
+import MenuItemComponent from 'components/GridTable/components/MenuItemComponent';
+import { IMenuItem } from 'components/GridTable/components/MenuItemComponent/types';
+import { useNestedMenuStyles } from 'components/GridTable/components/NestedMenu/styles';
+import { INestedMenuProps } from 'components/GridTable/components/NestedMenu/types';
+import _ from 'lodash';
```

Comment:
try not to use lodash where possible and if not please import the lodash method directly instead of all of lodash

Reply:
Done